base/src/main/java/com/smartdevicelink/proxy/rpc/TireStatus.java

```diff
@@ -145,90 +191,148 @@ public TireStatus(@NonNull WarningLightStatus pressureTellTale, @NonNull SingleT
     }
 
     /**
-     * @deprecated use {@link #setPressureTelltale(WarningLightStatus pressureTellTale)} instead.
+     * Sets the pressureTelltale.
+     *
+     * @param pressureTelltale Status of the Tire Pressure Telltale. See WarningLightStatus.
+     * @since SmartDeviceLink 2.0.0
      */
-    @Deprecated
-    public TireStatus setPressureTellTale(@NonNull WarningLightStatus pressureTellTale) {
-        return setPressureTelltale(pressureTellTale);
+    public TireStatus setPressureTellTale(WarningLightStatus pressureTelltale) {
+        setValue(KEY_PRESSURE_TELL_TALE, pressureTelltale);
+        return this;
     }
 
     /**
-     * @deprecated use {@link #getPressureTelltale()} instead.
+     * Gets the pressureTelltale.
+     *
+     * @return WarningLightStatus Status of the Tire Pressure Telltale. See WarningLightStatus.
+     * @since SmartDeviceLink 2.0.0
      */
-    @Deprecated
     public WarningLightStatus getPressureTellTale() {
-        return getPressureTelltale();
+        return (WarningLightStatus) getObject(WarningLightStatus.class, KEY_PRESSURE_TELL_TALE);
     }
 
     /**
-     * Sets the status of the tire pressure Telltale.
+     * Sets the leftFront.
      *
-     * @param pressureTellTale the status of the tire pressure Telltale.
+     * @param leftFront The status of the left front tire.
+     * @since SmartDeviceLink 2.0.0
      */
-    public TireStatus setPressureTelltale(@NonNull WarningLightStatus pressureTellTale) {
-        setValue(KEY_PRESSURE_TELL_TALE, pressureTellTale);
+    public TireStatus setLeftFront(SingleTireStatus leftFront) {
+        setValue(KEY_LEFT_FRONT, leftFront);
         return this;
     }
 
     /**
-     * Gets the status of the tire pressure Telltale.
+     * Gets the leftFront.
      *
-     * @return the status of the tire pressure Telltale.
+     * @return SingleTireStatus The status of the left front tire.
+     * @since SmartDeviceLink 2.0.0
      */
-    public WarningLightStatus getPressureTelltale() {
-        return (WarningLightStatus) getObject(WarningLightStatus.class, KEY_PRESSURE_TELL_TALE);
-    }
-
-    public TireStatus setLeftFront(@NonNull SingleTireStatus leftFront) {
-        setValue(KEY_LEFT_FRONT, leftFront);
-        return this;
-    }

```

Comment:
@kostyaBoss I would delete your `setPressureTellTale` and `getPressureTellTale` functions and replace it with what was there before:

```
/**
     * @deprecated use {@link #setPressureTelltale(WarningLightStatus pressureTellTale)} instead.
     */
    @Deprecated
    public TireStatus setPressureTellTale(@NonNull WarningLightStatus pressureTellTale) {
        return setPressureTelltale(pressureTellTale);
    }

    /**
     * @deprecated use {@link #getPressureTelltale()} instead.
     */
    @Deprecated
    public WarningLightStatus getPressureTellTale() {
        return getPressureTelltale();
    }

    /**
     * Sets the status of the tire pressure Telltale.
     *
     * @param pressureTellTale the status of the tire pressure Telltale.
     */
    public TireStatus setPressureTelltale(@NonNull WarningLightStatus pressureTellTale) {
        setValue(KEY_PRESSURE_TELL_TALE, pressureTellTale);
        return this;
    }

    /**
     * Gets the status of the tire pressure Telltale.
     *
     * @return the status of the tire pressure Telltale.
     */
    public WarningLightStatus getPressureTelltale() {
        return (WarningLightStatus) getObject(WarningLightStatus.class, KEY_PRESSURE_TELL_TALE);
    }
```

Once you do this, you shouldn't see the errors in the `RPCGenericTests`.  Please try this and let me know.